Mystery_Number/Mystery_Number.py

```diff
@@ -0,0 +1,39 @@
+'''
+    MYSTERY NUMBER
+
+    A Mystery Number is a number that can be expressed as a sum of two numbers,
+    where those two numbers are reverse of each other.
+'''
+
+'''  Approach : We iterate over from 1 to n/2, to get all the pairs of number, that
+                can sum up to the given no. n. and check if the numbers in the pair 
+                are reverse of each other.
+'''
+def rev(num) :
```

Comment:
I am not sure but I think, there is convention that no space before `:` in python.
Even in hash etc.